src/models/app.js

```diff
@@ -108,14 +108,12 @@ export default {
       return {
         ...state,
         isNavbar: true,
-        menuPopoverVisible: true,
```

Comment:
Fix the issue where the popup menu opens automatically in small screen.
<img width="676" alt="Screenshot 2025-02-10 at 3 11 05 PM" src="https://github.com/user-attachments/assets/3e9b6f0c-8a9a-413c-a1c4-22e36a9821ff" />